### PR TITLE
Feature/notification of active flight

### PR DESCRIPTION
**Pull request by Michael-AirMap**

### CODE REVIEW

File: Source/Traffic/TrafficService.swift [hunk 1]
```diff
@@ -116,12 +117,16 @@ internal class TrafficService: MQTTSessionDelegate {
...
+		let refreshCurrentFlightTimer = Observable<Int>.timer(0, period: 15, scheduler: MainScheduler.instance).mapToVoid()
 
-		refreshCurrentFlight
+		let refreshCurrentFlight = refreshCurrentFlightTimer
 			.filter {[unowned self] _ in AirMap.hasValidCredentials() && self.delegate != nil}
 			.skipWhile({[unowned self] _ in !AirMap.hasValidCredentials() || self.delegate == nil})
 			.flatMap(AirMap.rx.getCurrentAuthenticatedPilotFlight)
+			
+		Observable.merge(refreshCurrentFlight, receivedFlight.asObservable())
+			.debug("RECEIVED TRAFFIC"  )
```
Comment (adolfo) [inline Source/Traffic/TrafficService.swift]: Remove this debug

File: Source/Traffic/TrafficService.swift [hunk 2]
```diff
@@ -50,6 +50,7 @@ internal class TrafficService: MQTTSessionDelegate {
 	fileprivate var client = TrafficClient()
 	fileprivate var connectionState = Variable(ConnectionState.disconnected)
 	fileprivate var currentFlight = Variable(nil as AirMapFlight?)
+	fileprivate var receivedFlight = Variable(nil as AirMapFlight?)
```
Comment (adolfo) [inline Source/Traffic/TrafficService.swift]: Let's look at making this a `PublishSubject<AirMapFlight?>`, same with `currentFlight`

File: Source/Traffic/TrafficService.swift (hunk 1 above)
Comment (Michael-AirMap) [reply Source/Traffic/TrafficService.swift]: Removed

File: Source/Traffic/TrafficService.swift (hunk 2 above)
Comment (Michael-AirMap) [reply Source/Traffic/TrafficService.swift]: Why `PublishSubject<AirMapFlight?>`? `BehaviorRelay<AirMapFlight?>` would fit here more conveniently whereas `PublishSubject<AirMapFlight?>` would require more reworking

Comment (adolfo) [reply Source/Traffic/TrafficService.swift]: Do you need to keep it around as state or can it be transient in the observable chain?

Comment (Michael-AirMap) [reply Source/Traffic/TrafficService.swift]: `receivedFlight` can be transient. However, `currentFlight` is more embedded and requires state in a couple places

Comment (adolfo) [reply Source/Traffic/TrafficService.swift]: Ok then let's use `BehaviourRelay` and lose the deprecated `Variable`

**Pull request merged**